### PR TITLE
scripts/set-alias-page.py: add language option, fix script

### PR DESCRIPTION
**Pull request by sebastiaanspeck**
set-more-info-link will follow in another PR 😄

### CODE REVIEW

**Comment by vitorhcl** [comment]
> set-more-info-link will follow in another PR 😄 

I was wondering, these scripts and `set-page-title.py` share a lot of code, wouldn't it be a good idea to extract the common part to some functions, move them to another script (maybe one that starts with "_" to indicate it is not intended to be run directly) and then import them in each script?

This way would make changes that would require changing all scripts just because something "more fundamental" has changed much easier and also would make the creation of similar scripts much easier.

File: scripts/set-alias-page.py [hunk 1]
```diff
@@ -6,17 +6,19 @@
...
 
 Usage:
-    python3 scripts/set-alias-page.py [-p PAGE] [-s] [-S] [-n] [COMMAND]
+    python3 scripts/set-alias-page.py [-p PAGE] [-l LANGUAGE] [-s] [-S] [-n] [COMMAND]
 
 Options:
     -p, --page PAGE
         Specify the alias page in the format "platform/alias_command.md".
+    -l, --language LANGUAGE
+        Specify the language, in the format "fr" or "pt_BR", for which you want to sync or show any changes.
```
Comment (vitorhcl) [inline scripts/set-alias-page.py:21]:
```suggestion
        Specify the language, a POSIX Locale Name in the form of "<2_letter_language>[_<2_LETTER_CONTRY>]" (for example: "fr" or "pt_BR"), for which you want to sync or show any changes.
```

If you think this is too big/complex, I have another suggestions:

```suggestion
        Specify the language, in the form of "ll" or "ll_CC" (for example: "fr" or "pt_BR"), for which you want to sync or show any changes.
```

```suggestion
        Specify the language, in the form of "locale" or "locale_COUNTRY" (for example: "fr" or "pt_BR"), for which you want to sync or show any changes.
```

```suggestion
        Specify the language, a POSIX Locale Name in the form of "<language>[_<COUNTRY>]" (for example: "fr" or "pt_BR"), for which you want to sync or show any changes.
```

```suggestion
        Specify the language, a POSIX Locale Name in the form of "ll" or "ll_CC" (for example: "fr" or "pt_BR"), for which you want to sync or show any changes.
```

File: scripts/set-alias-page.py [hunk 2]
```diff
@@ -225,6 +233,14 @@ def main():
         default="",
         help='page name in the format "platform/alias_command.md"',
     )
+    parser.add_argument(
+        "-l",
+        "--language",
+        type=str,
+        required=False,
+        default="",
+        help='language in the format "fr" or "pt_BR"',
```
Comment (vitorhcl) [inline scripts/set-alias-page.py:242]:
```suggestion
        help='language in the format "ll" or "ll_CC" as in "fr" or "pt_BR"',
```
I know we should keep the help short, but saying examples are formats doesn't seem right to me. I think this is still short enough, what do you think?

**Review by vitorhcl — APPROVED** [review]
LGTM after we address my concerns. See each suggestion for more details.

**Review by kbdharun — APPROVED** [review]
LGTM, thanks for the additions. 

While testing on Windows (inside a Conda env), I noticed that encoding isn't specified in the functions using files leading to this error:

```ps
(myenv) PS C:\Users\kbdha\Desktop\tldr> python scripts/set-alias-page.py -S -l ta  
Traceback (most recent call last):
  File "C:\Users\kbdha\Desktop\tldr\scripts\set-alias-page.py", line 325, in <module>
    main()
    templates = get_templates(root)
  File "C:\Users\kbdha\Desktop\tldr\scripts\set-alias-page.py", line 88, in get_templates
    lines = f.readlines()
  File "E:\CondaInstall\envs\myenv\lib\encodings\cp1252.py", line 23, in decode  
    return codecs.charmap_decode(input,self.errors,decoding_table)[0]
UnicodeDecodeError: 'charmap' codec can't decode byte 0x8f in position 1196: character maps to <undefined>
```

Fixed it in https://github.com/tldr-pages/tldr/pull/12745/commits/cc35c495cb9fd9af10fd92330467d50e9b8f9028 commit.

---

Let's merge this PR for now and rebase #12755 to this.

**Pull request merged**